modules/ROOT/pages/getting-started.adoc

```diff
@@ -7,63 +7,93 @@ xref:glossary:index.adoc[glossary].
-
-All builds are performed by Tekton PipelineRuns within a namespace and usually driven by PipelineDefinitions
-committed to git repositories and triggered via https://pipelinesascode.com/[Pipelines as Code].
+In Kubernetes,
+link:https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/[namespaces]
+provide a foundational mechanism for isolating groups of resources within a
+single cluster. All of the {ProductName} resources and APIs that you interact
+with are scoped to namespaces, including your Components, Applications, Snapshots,
+Secrets and the Tekton PipelineRuns that perform builds, tests, and releases.
 
 ==== Tenant namespace
 These are namespaces where artifacts are produced from Tekton Pipelines. These can be accessed by more than one
-individual according to users' roles and the permissions defined by 
+individual according to their roles and the permissions defined by 
 link:https://konflux-ci.dev/architecture/ADR/0011-roles-and-permissions.html[those roles]. The tenant namespaces can be either
 for an individual or a team.
 //TODO: Document the process for getting access to/creating new namespaces
 
-You can grant, modify, or remove other users access to any namespace for which you have the `admin` role. Each
-namespace's admins have the responsibility to ensure that appropriate access is granted to users. If users are
-over-privileged, they may have access to any added secrets or to perform other unintended actions.
+In {ProductName}, you operate in a tenant namespace which is scoped to your
+team. One team can span multiple namespaces if needed, each with many
+applications with components. Namespaces should NOT be shared by many teams.
+
+In your tenant namespace, you will xref:building:creating.adoc[create
+Components and Applications], run the xref:building:index.adoc[build
+PipelineRuns] that are defined in your git repositories, view and iterate on
+the xref:testing:index.adoc[results of your IntegrationTestScenarios], and
+xref:releasing:index.adoc[create Releases] to release specific Snapshots
+according to specific ReleasePlans.
 
 ==== Managed namespace
-Sometimes users will need to perform actions that require credentials in someone else's control. Since admins
-in a tenant namespace can see all secrets, these actions need to be performed via separate managed namespaces.
-The primary interaction mode between tenant and managed namespaces is to create a Release referencing a specific
-Snapshot which will trigger a specific pipeline in the managed namespace.
+These are namespaces where a managed environment team manages release pipelines
+and credentials for your organization.
+
+The primary interaction mode between tenant and managed namespaces is to create
+a Release in a tenant namespace referencing a specific Snapshot which will
+trigger a specific release pipeline in the managed namespace. See
+xref:releasing:index.adoc[release documentation] for more.
+
+=== Custom Resource (CR)
```

Comment:
Tiny suggestion: since we're adding this CR section here, we could nest all the following sections under it, since they are all CRs as well (components, pipelines, application, etc). This would make it similar to what we did by adding  "Managed namespace" and "Tenant namespace" under "Namespaces".

Reply:
Done in 734d676 (I nested "build pipeline" under Component).